lms/views/lti/resource_link.py

```diff
@@ -31,7 +31,7 @@
     permission=Permissions.LTI_LAUNCH_ASSIGNMENT,
     schema=BasicLTILaunchSchema,
 )
-class BasicLaunchViews:
+class ResourceLinkLaunchViews:
```

Comment:
Renamed this view to use the LTI1.3 nomenclature like we do for deep linking (vs content item selection).

I guess this is entirely necessary but the name "basic launch" implies something spacial about them.